### ISSUE TITLE
Speakers don't seem to be enabled

### ISSUE THREAD

**Issue by retropele** [AsahiLinux/asahi-audio#27]
Hello, and sorry to bug the team.

My speakers on my M2 Macbook Pro 13'' don't seem to have been enabled. There are no output devices displayed, and I can't seem to do anything in Pulse Audio to enable them either.

Is there some way I can ensure the proper files etc. are installed? I am running Fedora Asahi 39, which I updated from 38 using sudo dnf upgrade --refresh some time ago.

Thank you :)

**Comment by chadmed** [comment]
Please run `asahi-diagnose` and attach the output here

**Comment by retropele** [comment]
> Please run `asahi-diagnose` and attach the output here

```Collecting system diagnostic information...

Saved diagnostic information to /home/***/asahi-diagnose-20231230-163053.txt

Pipewire macaudio profile is "none".
Headphones and speakers will not work. Select the "Default" or "HiFi" profile.```

**Comment by retropele** [comment]
Thanks to the diagnose function I found that asahi-audio isn't installed (obviously).

When I installed manually I had some conflicting packages (which I think were also pipewire/pulse audio) and was able to select allowerasing. Now the speakers are working and sound excellent! Just not sure why it didn't automatically install. Hopefully the log is at least useful to someone in fixing the bug.

This is fixed on my end now, I think, so thank you all very much for your hard work :)
